src/validation/ndrop/CMakeLists.txt

```diff
@@ -0,0 +1,145 @@
+set(NDROP_VALIDATION_DIR ${MAM_X_VALIDATION_DIR}/ndrop)
+set(NDROP_VALIDATION_SCRIPTS_DIR ${MAM_X_VALIDATION_DIR}/scripts)
+
+
+# These subdirectories contain Skywalker drivers for MAM4 parameterizations.
+# Include directory for .mod files.
+
+include_directories(${PROJECT_BINARY_DIR}/validation)
+
+# We use a single driver for all ndrop-related parameterizations.
+add_executable(ndrop_driver
+               ndrop_driver.cpp
+               ccncalc.cpp
+               ccncalc_single_cell.cpp
+               get_activate_frac.cpp
+               loadaer.cpp
+               activate_modal.cpp
+               explmix.cpp
+               maxsat.cpp
+               update_from_newcld.cpp
+               update_from_cldn_profile.cpp
+              #  update_from_explmix.cpp
```

Comment:
Is `update_from_explmix` still in progress? I noticed a comment in that file related to a thread team.

Reply:
The validation test is still in progress, but the subroutine was ported.  We could  remove this code from this PR.

Reply:
It's fine to leave the code in with this comment. I remember you and Mike telling me about this now.